### DISCUSSION TITLE
Is this project still alive? What to do about open PRs? Especially those that add lens calibrations?

### DISCUSSION THREAD

**Discussion by jonastr** [lensfun/lensfun#1526] [Q&A] [answered]
It seems this project is completely unmaintained. There are many, many open PRs open since months, some of them at least half a year. Many of them "only" add lens calibrations, so should be rather easy to review, I assume.

What can we do to change this really unfortunate state?

**Comment by Sturmflut** [comment]
The situation has been improving recently. Since the beginning of the year, more than one issue per day has been closed on average. The most pressing issues (e.g. half of the Nikon mirrorless camera lineup missing) have been addressed, and the lenses that have been uploaded most often have been worked on.

In many cases we have to coordinate lens model names with the exiv2 project or find out why lenses aren't being detected. Of course we also want to check submitted calibrations against actual images. Many issues/uploads and PRs require communication with the authors/uploaders, and it's a bit hard to know who is just waiting to answer and who cannot be contacted anymore. In general it's often not as easy as it seems. I've spent the last two days just finding out how to get three missing Sigma lenses into exiv2.

If there is something you want to have prioritised, please just comment on the PR or issue.

**Comment by tuxfanx** [comment]
I also had the impression that the project was slowly dying. That's why I offered my help in processing the calibration requests at the beginning of the year ( https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst ). In the meantime I have been able to become a member of the calibration group, but no one has been able to give me access to the images in the cloud. Maybe this way I can reach someone who can help.

**Reply by tuxfanx** [reply]
...got access to the cloud. Thanks for the support to sarunasb, Sturmflut and bronger.

**Comment by Sturmflut** [comment, marked as answer]
More than 100 issues and pull requests have been closed since Christmas, and work on new releases and website updates has picked up considerably. I think this question is answered :)

**Reply by jonastr** [reply]
Great to hear that and thanks everyone involved for your work & dedication! 👍🏻
